main.go

```diff
@@ -52,7 +52,13 @@ var (
 
 func main() {
 	nodeMetadata := make(map[string]string)
-	flag.CommandLine.Var(newStringMapVal(&nodeMetadata), "node-metadata-experimental", "additional metadata of the form key=value to be included in the node configuration. This flag is EXPERIMENTAL and may be changed or removed in a later release.")
+	flag.Var(newStringMapVal(&nodeMetadata), "node-metadata",
+		"additional metadata of the form key=value to be included in the node configuration")
+
+	flag.Var(flag.Lookup("secrets-dir").Value, "secrets-dir-experimental",
+		"alias of secrets-dir. This flag is EXPERIMENTAL and will be removed in a later release")
+	flag.Var(flag.Lookup("node-metadata").Value, "node-metadata-experimental",
```

Comment:
TIL about flag name aliasing in Go. Thanks.

Reply:
Yeah, I think you could do it with just the like of `StringVar`, but then you have to redefine the default value. Some iteration got me here, which feels pretty solid.